deployment-scripts/jobs-deployment/tests/test_deploy_job.py

```diff
@@ -9,11 +9,11 @@
 import boto3
 from moto import mock_s3
 
-sys.path.insert(0, "../")
```

Comment:
@grzegorz8 Let me know if the tests are running for You as they are now after my changes in the imports.

Reply:
Please uncomment